index.html

```diff
@@ -145,9 +146,39 @@ <h3>Multi-Post Stories Gain+Glory</h3>
             <li>html</li>
           </ul>
           <button type="button" class="btn-card">See Project</button>
-        </div>
-      </div>
+        </li> -->
+      </ul>
     </section>
+
+    <article class="flex-center">
+      <!-- <div class="card-popup"> -->
+      <!-- <div class="project-img">
+        <img class="card-img" src="./images/Snapshoot-Portfolio-mobile@2x.png" alt="">
+          <img class="hide-icon" src="./images/icons/Disabled.svg" alt="">
+        </div>
+        <div class="article">
+          <h3>Keeping track of hundreds of components_1</h3>
+          <ul class="program-lang d-flex">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li> -->
+      <!-- <li>html</li> -->
+      <!-- </ul>
+          <p>Project_1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+            the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+            galley of type and scrambled it 1960s with the relea</p>
+          <div class="btns-popup d-flex flex-center">
+            <button type="button" class="btn-card d-flex flex-center">See Live <img
```

Comment:
- [ ] Please follow the **Dry rule** for the HTML, do not leave any commented codes in your project.

Reply:
thanks